### DISCUSSION TITLE
ScriptScraperGraph with Ollama models

### DISCUSSION THREAD

**Discussion by poying** [ScrapeGraphAI/Scrapegraph-ai#841] [Q&A]
Does that mean we can't use Ollama models with ScriptScraperGraph because we must specify the LLM format as json?

The example doesn't work https://github.com/ScrapeGraphAI/Scrapegraph-ai/blob/main/examples/local_models/script_generator_ollama.py

**Comment by VinciGit00** [comment]
Can you open the issue where you show the output?
